docs/spec/special-types.rst

```diff
@@ -47,7 +47,8 @@ to ``tuple[Any, ...]``. As well, a bare
 
 ``Any`` can also be used as a base class. This can be useful for
 avoiding type checker errors with classes that can duck type anywhere or
-are highly dynamic.
+are highly dynamic. When ``Any`` is present in the bases of a type,
```

Comment:
nit: would this be slightly clearer as "is present in the MRO of a type"?

rationale: consider `class AnySub(AnyFirst): pass`, which does not have `Any` in its bases, but does have `Any` in its MRO. I believe this new specification is also intended to apply to `AnySub`, just as it does to `AnyFirst`.

It may make sense to also clarify this by adding `AnySub` as an example in the conformance suite.

Reply:
Good points on being clearer, I've gone ahead and adjusted this accordingly.